package.json

```diff
@@ -56,6 +55,7 @@
     "eslint-plugin-react-hooks": "^4.1.2",
     "husky": "^8.0.1",
     "jest": "^28.0.3",
+    "jest-environment-jsdom": "^28.1.0",
```

Comment:
![Screenshot 2022-05-19 at 16 59 55](https://user-images.githubusercontent.com/14129033/169345385-e467d544-9ca1-4b25-9053-c4de8ecca64c.png)

You need to install `jest-environment-jsdom` since `jest` v28 apparently